lib/fastlane/plugin/wpmreleasetoolkit/versioning/formatters/derived_build_code_formatter.rb

```diff
@@ -3,23 +3,34 @@
 module Fastlane
   module Wpmreleasetoolkit
     module Versioning
+      MAX_TOTAL_DIGITS = 8
```

Comment:
💄 Adding comment for extra context
```suggestion
       # Max total for `*_digits` params, not counting prefix
      MAX_TOTAL_DIGITS = 8
```